tests/Rules.Framework.IntegrationTests/Scenarios/Scenario8/TexasHoldEmPokerSingleCombinationsTests.cs

```diff
@@ -0,0 +1,58 @@
+namespace Rules.Framework.IntegrationTests.Scenarios.Scenario8
+{
+    using System;
+    using System.Threading.Tasks;
+    using FluentAssertions;
+    using Rules.Framework.BenchmarkTests.Tests.Benchmark3;
+    using Rules.Framework.IntegrationTests.Common.Scenarios;
+    using Rules.Framework.Providers.InMemory;
+    using Xunit;
+
+    public class TexasHoldEmPokerSingleCombinationsTests
+    {
+        [Theory]
+        [InlineData(false)]
+        [InlineData(true)]
+        public async Task PokerCombinations_Given_EvaluatesStraightCombination(bool enableCompilation)
+        {
+            // Arrange
+            var matchDate = new DateTime(2023, 1, 1);
+            var conditions = new[]
+            {
+                new Condition<ConditionTypes> { Type = ConditionTypes.NumberOfKings, Value = 1 },
```

Comment:
You can use the Condition ctor to make it cleaner.

Reply:
I did change it to follow your recommendation 👌

Reply:
Speaking of which, what do you think about making the parameterless constructor Obsolete?

Reply:
Actually I've done it here: https://github.com/Farfetch/rules-framework/pull/146 

You can either merge this with your request or we'll evaluate this MR separately.

Reply:
Then let's use your implementation, my changes where done on another branch that still doesn't have a PR open. Go ahead, I'll merge yours into that branch 👌

Reply:
What you're saying is that https://github.com/Farfetch/rules-framework/pull/146 should be merged, right? Can you approve please?

Reply:
Yes, it should be merged 👍 if there are no more corrections to be done on this branch, we could merge yours instead and close this PR.